Mutagen.Bethesda.Analyzers.Testing/ParamExtensions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Mutagen.Bethesda.Analyzers.SDK.Analyzers;
+using Mutagen.Bethesda.Plugins.Records;
+
+namespace Mutagen.Bethesda.Analyzers.Testing
+{
+    public static class ParamExtensions
```

Comment:
Should maybe go into the `TestingUtils` project which we can publish on nuget for others to use

Reply:
Do we want to just merge the `Mutagen.Bethesda.Analyzers.TestingUtils` project with `Mutagen.Bethesda.Analyzers.Testing`?  Seems to have the same goal